Studio-Lovelies/GG-JointJustice-Unity
# Change `Evidence` to `EvidenceData` to be in line with `Actor`/`ActorData`

**This is a proactive proposal PR, so I'm happy to close it if we don't agree on it!**

Opening it up to discussion or creating an Issue before seeing the code changes for it didn't really feel worth it, given the small size of the change.

## Summary

Writing [the word list for this project](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/wiki/Word-List), I realized [our definitions of Actors and Evidence are not aligned](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/wiki/Word-List#:~:text=Actor,Evidence%20object%20in%20Unity.%22), but realized, they might as well:
- `Actor`: Thing we instantiate in the game
- `ActorData`: ScriptableObject of data for Actors

...which would dictate that...

- `Evidence`: Thing we instantiate in the game
- `EvidenceData`: ScriptableObject of data for Evidence

However:
- we don't instantiate Evidence, only UI renderers for it (so there's technically no need for an `Evidence` class)
- what we should call `EvidenceData` is actually called `Evidence` right now

This PR changes `Evidence` to `EvidenceData`, unless we're referring to "a piece of evidence" from a gameplay-perspetive inside the comments.

## Testing instructions
1. Check if there are any missed instances of `Evidence` that should be changed and aren't part of this PR
2. Run the game and make sure scenes run as expected (I used the `Game` scene with the `RossCoolX` story.)

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
